source/as-promise/types.ts

```diff
@@ -72,12 +72,101 @@ export interface Hooks extends RequestHooks {
 
 export interface PaginationOptions<T, R> {
 	pagination?: {
+		/**
+			A function that transform [`Response`](#response) into an array of items.
+			This is where you should do the parsing.
+
+			@default response => JSON.parse(response.body)
+			*/
```

Comment:
```suggestion
		/**
		A function that transform [`Response`](#response) into an array of items.
		This is where you should do the parsing.

		@default response => JSON.parse(response.body)
		*/
```

Same everywhere else.